packages/diagrams/lib/src/icon.dart

```diff
@@ -20,15 +22,32 @@ class IconDiagram extends StatelessWidget implements DiagramMetadata {
   Widget build(BuildContext context) {
     return ConstrainedBox(
       key: UniqueKey(),
-      constraints: BoxConstraints.tight(const Size(140.0, 140.0)),
+      constraints: BoxConstraints.tight(const Size(200.0, 100.0)),
       child: Container(
-        alignment: FractionalOffset.center,
         padding: const EdgeInsets.all(5.0),
         color: Colors.white,
-        child: const Icon(
-          Icons.add,
-          color: Colors.pink,
-          size: 30.0,
+        child: Center(
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: const <Widget>[
+              Icon(
+                Icons.favorite,
+                color: Colors.pink,
+                size: 24.0,
+                semanticLabel: 'Text to announce in accessibility modes',
```

Comment:
This is great to include in the sample code for the docs, but we don't need it here for the static image generator.